components/organisms/Repositories/repositories.tsx

```diff
@@ -10,19 +11,32 @@ import TableHeader from "components/molecules/TableHeader/table-header";
 
 import { useRepositoriesList } from "lib/hooks/useRepositoriesList";
 import useSupabaseAuth from "lib/hooks/useSupabaseAuth";
+import useStore from "lib/store";
+import useFilterRepos from "lib/hooks/useFilterRepos";
 
 import RepositoriesTable, { classNames } from "../RepositoriesTable/repositories-table";
 import RepoNotIndexed from "./repository-not-indexed";
-import useStore from "lib/store";
 
 interface RepositoriesProps {
   repositories?: number[];
 }
 
+type FilterOptions = keyof DbRepo | "prsCount" |"activity"
+
+const renderArrow = (order: string | undefined) => {
+  return order === "ASC" ? <FaArrowUp className="text-light-slate-11 ml-2" fontSize={16} />
```

Comment:
can you please create an enum `... enum { ASC="ASC"}`

Reply:
Could you expand on that? 
I'm not sure I understand

Reply:
`enum Order {
  ASC= 'ASC',
  DESC="DESC"
} `
And here you have to change in 
`return order === Order.ASC ? <FaArrowUp className="text-light-slate-11 ml-2" fontSize={16} />`